service_mesh/v2x/ossm-create-smcp.adoc

```diff
@@ -10,7 +10,12 @@ You can deploy a basic installation of the `ServiceMeshControlPlane` by using ei
 
 [NOTE]
 ====
```

Comment:
This might be out of scope for the PR, but I suggest putting (SMCP) after the first instance of `ServiceMeshControlPlane` on this page to define the acronym, as it is used interchangeably in this section of content.